### PR TITLE
[Docs] Reformat op docs

### PR DESCRIPTION
**Pull request by imabackstabber**
## Motivation

Reformat op docs.Add devices information.

## Modification

Reformat `ops.md`

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects, like MMDet or MMCls.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Comment by teamwong111** [comment]
Please Fix the lint. And tables of data types (float, half) will not be provided, right?

**Comment by imabackstabber** [comment]
> Please Fix the lint. And tables of data types (float, half) will not be provided, right?

lint has been fixed.We don't plan to offer information about half type yet.

File: docs/en/understand_mmcv/ops.md [hunk 1]
```diff
@@ -1,47 +1,59 @@
...
+| ActiveRotatedFilter          | √   | √    |     |
+| AssignScoreWithK             |     | √    |     |
+| BallQuery                    |     | √    |     |
+| BBoxOverlaps                 |     | √    | √   |
+| BorderAlign                  |     | √    |     |
+| BoxIouRotated                | √   | √    |     |
+| CARAFE                       |     | √    |     |
+| ChamferDistance              |     | √    |     |
+| CrissCrossAttention          |     | √    |     |
+| ContourExpand                |     |      |     |
```
Comment (teamwong111) [inline docs/en/understand_mmcv/ops.md:16]:
```suggestion
| ContourExpand                | √   |      |     |
```

File: docs/en/understand_mmcv/ops.md [hunk 2]
```diff
@@ -1,47 +1,59 @@
...
+| DynamicScatter               |     | √    |     |
+| FurthestPointSample          |     | √    |     |
+| FurthestPointSampleWithDist  |     | √    |     |
+| FusedBiasLeakyrelu           |     | √    |     |
+| GatherPoints                 |     | √    |     |
+| GroupPoints                  |     | √    |     |
+| Iou3d                        |     | √    |     |
+| KNN                          |     | √    |     |
+| MaskedConv                   |     | √    |     |
+| MergeCells                   |     |      |     |
```
Comment (teamwong111) [inline docs/en/understand_mmcv/ops.md:32]: looks strange that no device supports it.

Comment (imabackstabber) [reply docs/en/understand_mmcv/ops.md]: > looks strange that no device supports it.

Has been fixed.

**Review by teamwong111 — APPROVED** [review]

**Review by grimoire — APPROVED** [review]

**Review by zhouzaida — APPROVED** [review]

**Pull request merged**